runtime/testy/Ansible.py

```diff
@@ -3,9 +3,19 @@
 import time
 import sys
 import selectors
+import runtime_pb2
+import ansible_pb2
 
 from runtimeUtil import *
 
+stateToEnum = {0:runtime_pb2.RuntimeData.STUDENT_CRASHED,
```

Comment:
This dict is literally a no-op. `runtime_pb2.RuntimeData.STUDENT_CRASHED` should already be an int. Same with the rest of the values here.

See also `runtime_pb2.RuntimeData.State.items()`, which actually contains a list of (string, int) key-value pairs.